great_expectations/data_context/store/store.py

```diff
@@ -218,7 +218,7 @@ def get_all(self) -> list[Any]:
 
         return list(map(self.deserialize, objs))
 
-    def set(self, key: DataContextKey, value: Any, **kwargs) -> None:
+    def set(self, key: DataContextKey, value: Any, **kwargs) -> Any:
```

Comment:
Do we know what this might be? More for my own edification I think we're gonna probably redo stores at some point so doubt we need to get super granular

Reply:
I think it varies a lot. Sometimes `None`, sometimes `bool`, sometimes a cloud identifier, etc. I hate using `Any`, but I think this is the most honest return type at this point